tool/colors/update_colors.dart

```diff
@@ -20,62 +23,40 @@ void main(List<String> args) async {
     exit(1);
   }
 
-  // TODO(dantup): Split into two scripts instead of using a flag? It's to allow
-  // easier debugging of the second step.
-  if (args.contains('--generate-properties')) {
-    print('Generating property files:');
-    generatePropertiesFiles();
-
-    exit(0);
-  } else {
-    print('Generating dart files:');
-    await generateDartFiles();
-
-    // Re-run the script with `--generate-properties` so it can load the files
-    // we generated.
-    final results = Process.runSync(Platform.resolvedExecutable,
-        [Platform.script.toFilePath(), '--generate-propertes']);
-    stdout.write(results.stdout);
-    stderr.write(results.stderr);
-
-    exit(0);
-  }
+  print('Generating dart files:');
+  await generateDartFiles();
 }
 
-Future generateDartFiles() async {
-  // TODO: Use the files from the local flutter checkout instead of a download
```

Comment:
This todo is likely worth retaining - I think we're better off avoiding downloads when re-generating these files if we can avoid them.

Reply:
Ah yes, I slightly misread it thinking it was about having two downloaded copies. I've added this back.